exercises/K-functions-parameters/exercise5.js

```diff
@@ -1,5 +1,9 @@
 // Declare your function here
-
+function createLongGreeting(string, number) {
+    return `Hello, my name is ${string} and I'm ${number} years old`
```

Comment:
The semicolon is missing from the end of the command. It is not mandatory in JS but it's better to use it.